Aztec/Classes/TextKit/MediaAttachment.swift

```diff
@@ -334,8 +334,9 @@ private extension MediaAttachment {
     /// Requests a new asset (asynchronously), and on completion, triggers a relayout cycle.
     ///
     private func updateImage(in textContainer: NSTextContainer?) {
-
-        self.image = delegate!.mediaAttachmentPlaceholderImageFor(attachment: self)
+        if mustDisplayPlaceholder {
```

Comment:
Why not avoid the method and the conditional block completely and just use the guard below to display the placeholder?

The execution path when `url == nil` in this method is a bit weird.